corehq/apps/locations/models.py

```diff
@@ -281,15 +281,13 @@ def couch_locations(self, wrapped=True):
         return locations
 
     def accessible_to_user(self, domain, user):
-        from corehq.apps.locations.util import get_locations_and_children
-
         if user.has_permission(domain, 'access_all_locations'):
             return self.all()
 
         assigned_location_ids = user.get_location_ids(domain)
         if not assigned_location_ids:
-            return self.none()  # No locations are accessible to this user
-        return self.all() & get_locations_and_children(assigned_location_ids)
+            return self.none()  # No locations are assigned to this user
+        return self.all() & SQLLocation.active_objects.get_locations_and_children(assigned_location_ids)
```

Comment:
`accessible_to_user` is in `LocationQueriesMixin`, so it's available as a queryset method AND as a manager method.  `get_locations_and_children`, conversely, is only available as a manager method, so it's not guaranteed to be available (it's only defined in the subclass).

`get_locations_and_children` needs to be a manager method under it's current implementation.  We could make it a queryset method if we adopted the strategy of `self.all() & ...` like is used for `accessible_to_user`, but it makes more sense to me as a method which *returns* a queryset (ie a manager method) than as a method which *filters* a queryset (a queryset method).